README.md

```diff
@@ -5,11 +5,13 @@ A capstone project for UChicago's CAPP122 where abortion related data is taken f
 
 ## Project Requirements
 - [Poetry](https://python-poetry.org/docs/basic-usage/)
+- An API key from [Abortion Policy API](https://www.abortionpolicyapi.com/) stored as an environment variable called `ABORTION_POLICY_API_KEY`.
+  - The developers of this project stored it in our `.zshrc` file, but you may opt for an `.env` file if that is more comfortable.
 
```

Comment:
Might be good to also add that individuals who want to recreate this project would have to request access to the ANSIRH abortion facility database through this website: https://abortionfacilitydatabase-ucsf.hub.arcgis.com/